File: el_back.models/Employee.cs
using System;
using System.Collections.Generic;

namespace el_back.models
{
    public class Employee
    {
        public long Id { get; set; }
        public string FirstName { get; set; }
        public string LastName { get; set; }
        public IEnumerable<long> Jobs { get; set; }
        public DateTime EmploymentDate { get; set; }
        public long Rate { get; set; }
        public DateTime CreationDate { get; set; }
    }
}

File: el_front/src/units/employee/interfaces/employee.ts
import { Job } from './job';

export interface Employee {
  id: number;
  firstName: string;
  lastName: string;
  jobs: Job[];
  employmentDate: string; // DateTime - maybe, it must be as part of Job model
  rate: number;
  creationDate: string;   // DateTime
}

export interface EmployeeFormModel {
  firstName: string;
  lastName: string;
  employmentDate: string;
  rate: number;
  jobs: number[];
}
